client/purplecaffeine/core.py

```diff
@@ -245,6 +254,34 @@ def get(self, trial_id: str) -> Trial:
 class ApiBackend(BaseBackend):
     """API backend class."""
 
+    def __init__(self, username: str, password: str, host: str):
+        """Creates backend for APIServer.
+
+        Example:
+            >>> backend = ApiBackend(
+            >>>     host="http://localhost:8000/",
+            >>>     username="admin",
+            >>>     password="123"
+            >>> )
+
+        Args:
+            username: username
+            password: password
+            host: host of api server
+        """
+        self.username = username
+        self.host = host
```

Comment:
Host is also in the Configuration file :
https://github.com/IceKhan13/purplecaffeine/blob/f7bdf361b5c20f90a0af2562ff1cd138581d8cfa/client/purplecaffeine/helpers/conf.py#L10

Reply:
we will not use configuration files as each deployment will have their host in different place

Reply:
We should allow the people to select host they want and if they don't set anything just use the one from the Configuration

Reply:
@IceKhan13 what do you mean they all gonna be different ?
We will always have only 1 simple url to call the api. The main DNS, not gonna change, or am I missing something ?